cobalt/dom/source_buffer.cc

```diff
@@ -170,9 +181,19 @@ SourceBuffer::SourceBuffer(script::EnvironmentSettings* settings,
       chunk_demuxer_(chunk_demuxer),
       event_queue_(event_queue),
       audio_tracks_(
-          new AudioTrackList(settings, media_source->GetMediaElement())),
+          IsMediaElementUsingMediaSourceAttachmentMethodsEnabled(
+              environment_settings())
```

Comment:
Would it make more sense to initialize this as a variable `is_using_media_source_attachment_methods_`, and replace it with the calls on all other places?